src/Livewire/Forms/PriceListForm.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace FluxErp\Livewire\Forms;
+
+use FluxErp\Actions\FluxAction;
+use FluxErp\Actions\PriceList\CreatePriceList;
+use FluxErp\Actions\PriceList\DeletePriceList;
+use FluxErp\Actions\PriceList\UpdatePriceList;
+use FluxErp\Models\PriceList;
+use Livewire\Attributes\Locked;
+
+class PriceListForm extends FluxForm
+{
+    #[Locked]
+    public ?int $id = null;
+
+    public ?int $parent_id = null;
+
+    public ?string $name = null;
+
+    public ?string $price_list_code = null;
+
+    public ?string $rounding_method_enum = null;
+
+    public ?int $rounding_precision = null;
+
+    public ?int $rounding_number = null;
+
+    public ?string $rounding_mode = null;
+
+    public ?bool $is_net = true;
+
+    public ?bool $is_default = false;
+
+    public ?array $discount = [
```

Comment:
can this be null?

Reply:
not really, I made this nullable, because it was easier to circumvent the issue if a price list has no discount. Otherwise I would get an exception during fill() method.